packages/backend/vite.config.js

```diff
@@ -53,6 +55,35 @@ const config = {
       output: {
         entryFileNames: '[name].cjs',
       },
+      plugins: [
+        {
+          // copy the swagger-ui-dist files to the dist folder as we need the files to be served
+          name: 'copy-swagger-ui',
+          async buildStart() {
+            const start = performance.now();
+            const source = resolve('../../node_modules/swagger-ui-dist');
+            const destination = resolve('dist/swagger-ui');
+
+            // Ensure destination directory exists
+            if (!existsSync(destination)) {
+              await mkdir(destination, { recursive: true });
+            }
+
+            // Copy files
+            const entries = await readdir(source, { withFileTypes: true });
+
+            for (const entry of entries) {
+              const srcFile = join(source, entry.name);
+              const destFile = join(destination, entry.name);
+
+              if (entry.isFile()) {
+                await copyFile(srcFile, destFile);
+              }
```

Comment:
why coping only the files ? can't we just copy the full folder using [fs.promises.cp](https://nodejs.org/api/fs.html#fspromisescpsrc-dest-options) and use recursive option?

Reply:
different attempt

I switched to copy and added a filter to not copy the .map files